### DISCUSSION TITLE
`include_docs` is invalid for reduce

### DISCUSSION THREAD

**Discussion by kasgithub8** [Budibase/budibase#8636] [Help]
Hey,

every time I get to the homepage of my app, the error `include_docs is invalid for reduce` shows.
![Screenshot 2022-11-10 164545](https://user-images.githubusercontent.com/95284278/201140984-cbc9e397-14f7-4ffc-9fa1-ffeed7c6daf3.png)
This is what the network tab of that page looks like.

Any help is appreciated!

**Comment by melohagan** [comment]
Hey @kasgithub8 

Are you able to share your app export please? You can send it to support@budibase.com if you'd prefer

**Reply by kasgithub8** [reply]
This is what happens when I'm trying to export: 
<img width="455" alt="image" src="https://user-images.githubusercontent.com/95284278/201304325-0ae907a4-b8ed-4093-a331-b2168c616a25.png">

**Reply by melohagan** [reply]
What do you see in the console/network tab when you try to export?

**Reply by kasgithub8** [reply]
This:
<img width="413" alt="image" src="https://user-images.githubusercontent.com/95284278/201305843-7064316b-48c6-4ecc-bd2d-771e2f97b6ac.png">

**Reply by melohagan** [reply]
Anything in the network tab?

**Reply by kasgithub8** [reply]
Nothing

**Reply by melohagan** [reply]
Would you mind sharing your tenant-id (your Budibase subdomain) so that we can investigate further please? 
You can email it to support@budibase.com if you'd rather not share it here.

**Reply by kasgithub8** [reply]
Tenant-ID is: bapps

**Reply by melohagan** [reply]
Hey @kasgithub8 

Could you please do the following:

- open dev tools
- open network
- use the 🚫 symbol to clear the logs
- ensure the All filter is on
- reload the app

then please take a screenshot of the entire panel contents.

**Reply by melohagan** [reply]
On the homepage. Is that within the client app, or the design section of the builder?

**Reply by kasgithub8** [reply]
> then please take a screenshot of the entire panel contents.

<img width="592" alt="image" src="https://user-images.githubusercontent.com/95284278/201315480-a669f14f-c8fe-4838-b497-ea47e18cd391.png">

**Reply by kasgithub8** [reply]
> On the homepage. Is that within the client app, or the design section of the builder?

Both, just the "/" screen.

**Reply by melohagan** [reply]
Hey @kasgithub8 I think you forgot to reload the app. There should be some network requests in the screenshot.

**Reply by kasgithub8** [reply]
> Hey @kasgithub8 I think you forgot to reload the app. There should be some network requests in the screenshot.

Was I meant to do more than F5? Do I need to reload the app from the builder?

**Reply by melohagan** [reply]
No a refresh of the page was correct. Hmm.

**Reply by kasgithub8** [reply]
> No a refresh of the page was correct. Hmm.

Hi,

there was another error that included one of my views. I re-added the data provider for that on my "/" screen and it did the trick. I don't know why tho.
Still can't export tho, is that a backup pro feature or why is "/backups/" in the link?
<img width="168" alt="image" src="https://user-images.githubusercontent.com/95284278/201322596-e47538a9-e785-4091-b95c-40d97b256f90.png">

**Reply by kasgithub8** [reply]
@melohagan it doesn't stop...
I'm just trying to update a row.
<img width="359" alt="image" src="https://user-images.githubusercontent.com/95284278/201680134-38696149-fdb3-411e-a976-4b69b9489e56.png">
